llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2849,7 +2849,7 @@ bool IRTranslator::translateInvoke(const User &U,
     return false;
 
   // FIXME: support whatever these are.
-  if (I.countOperandBundlesOfType(LLVMContext::OB_deopt))
```

Comment:
Does this imply there could have been multiple?

Reply:
Actually, it doesn't, because the instruction can only have one deopt bundle:
https://llvm.org/docs/LangRef.html#deoptimization-operand-bundles

> There can be at most one "deopt" operand bundle attached to a call site